src/main/java/com/uber/cadence/internal/worker/WorkflowWorker.java

```diff
@@ -240,14 +240,14 @@ private void sendReply(
 
   private class DecisionTaskWithHistoryIteratorImpl implements DecisionTaskWithHistoryIterator {
 
-    private final Duration RetryServiceOperationInitialInterval = Duration.ofMillis(200);
-    private final Duration RetryServiceOperationMaxInterval = Duration.ofSeconds(4);
-    private final Duration PaginationStart = Duration.ofMillis(System.currentTimeMillis());
-    private Duration DecisionTaskStartToCloseTimeout = Duration.ofSeconds(60);
+    private final Duration retryServiceOperationInitialInterval = Duration.ofMillis(200);
+    private final Duration retryServiceOperationMaxInterval = Duration.ofSeconds(4);
+    private final Duration paginationStart = Duration.ofMillis(System.currentTimeMillis());
+    private Duration decisionTaskStartToCloseTimeout = null;
 
     private final Duration RetryServiceOperationExpirationInterval() {
```

Comment:
Method name should start from a lower case letter.

Reply:
resolved